odinmain/src/main/kotlin/me/odinmain/features/impl/skyblock/EnrageDisplay.kt

```diff
@@ -0,0 +1,57 @@
+package me.odinmain.features.impl.skyblock
+
+import me.odinmain.events.impl.ServerTickEvent
+import me.odinmain.features.Category
+import me.odinmain.features.Module
+import me.odinmain.features.settings.impl.HudSetting
+import me.odinmain.font.OdinFont
+import me.odinmain.ui.hud.HudElement
+import me.odinmain.utils.render.Color
+import me.odinmain.utils.render.getTextWidth
+import me.odinmain.utils.render.text
+import me.odinmain.utils.skyblock.itemID
+import me.odinmain.utils.skyblock.modMessage
+import net.minecraft.network.play.server.S29PacketSoundEffect
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object EnrageDisplay : Module (
+    name = "Enrage Display",
+    description = "Timer for cooldown of reaper armor enrage",
+    category = Category.SKYBLOCK
+) {
+    private val hud: HudElement by HudSetting("Enrage Timer Hud", 10f, 10f, 1f, true) {
+        if (it) {
+            text("§4Enrage: §a119t", 1f, 9f, Color.RED, 12f, OdinFont.REGULAR, shadow = true)
+            getTextWidth("Enrage: 119t", 12f) + 2f to 16f
+        } else {
+            if (enrageTime.time >= 60) {
+                text("§4Enrage: §a${enrageTime.time}t", 1f, 9f, Color.WHITE, 12f, OdinFont.REGULAR, shadow = true)
+            }
+            if (enrageTime.time in 30..60) {
```

Comment:
Make these into else if's just because its cleaner, and that is the functionality you want from the code.

Reply:
Actually just make these into a when to get the color code that should be used;
val colorCode = when () {
  enrageTime.time >= 60 -> "§a"
  enrageTime.time in 30..60 -> "§e"
  enrageTime.time in 0..30 -> "§c"
  else -> return
}
text("§4Enrage: ${colorCode}${enrageTime.time}t", 1f, 9f, Color.WHITE, 12f, OdinFont.REGULAR, shadow = true)